src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/registries/FieldRegistry.js

```diff
@@ -21,6 +21,10 @@ class FieldRegistry {
     }
 
     get(name: string) {
+        if (!(name in this.fields)) {
```

Comment:
I remember we've been talking about this. But please also add a test for that :-)

Reply:
Test is added